README.md

```diff
@@ -20,6 +22,32 @@ jobs:
 
 No further action is required.
 
+## Setup using custom token
+
+This approach uses a custom token that you provide. This is necessary in order to trigger GitHub actions CI from a tagbot tag, in order to build documentation for tagged versions of your package.
+
+**Step 1:** Generate a GitHub personal access token. To do so, go to <https://github.com/settings/tokens/new>, name it anything you wish, and select either `repo` or `public_repo`, depending on whether or not you will use tagbot with private repositories. Save this somewhere secret, like a password, so you can reuse the same token for each tagbot workflow deployment you make.
```

Comment:
> Save this somewhere secret, like a password, so you can reuse the same token for each tagbot workflow deployment you make.

I would suggest that, just like passwords, you use a unique token for each repo, or at least one for each organisation. In case your token gets compromised, you only have to generate a new one for the affected repos which could save you a lot of trouble.

Reply:
Agree. Done.

Reply:
That was quick :D